contrib/babelfishpg_tsql/src/rolecmds.c

```diff
@@ -2156,6 +2156,7 @@ babelfish_add_domain_mapping_entry_internal(PG_FUNCTION_ARGS)
 		MemoryContext ectx;
 		ErrorData  *edata;
 
+		HOLD_INTERRUPTS();
 		ectx = MemoryContextSwitchTo(ccxt);
```

Comment:
while operation required hold/resume here?

Reply:
Good catch! it is not needed here.